packages/shared/src/components/PostOptionsMenu.tsx

```diff
@@ -495,11 +500,51 @@ export default function PostOptionsMenu({
   postOptions.push({
     icon: <MenuIcon Icon={BlockIcon} />,
     label: isSourceBlocked
-      ? `Show posts from ${post?.source?.name}`
-      : `Don't show posts from ${post?.source?.name}`,
+      ? `Unblock ${post?.source?.name}`
+      : `Block ${post?.source?.name}`,
```

Comment:
Feels like the `Block` can be misleading that it will block the source across the board. Maybe saying "Block Source from this Feed" or something similar. Either way, this is non-blocking. Just some thought about the new labe.

Reply:
This change comes from the slack thread in description, but feel free to propose some changes in slack and I can change in minutes ✔️